services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java

```diff
@@ -194,6 +194,7 @@ private static Map<String, Object> initAttributes(KeycloakSession session, Realm
         String errorMessage = messagesBundle.getProperty(errorKey);
 
         attributes.put("message", new MessageBean(errorMessage, MessageType.ERROR));
+        attributes.put("darkMode", true);
```

Comment:
This is when an error is shown, there is no realm and there are no properties so we ignore the setting

Reply:
I see we're reading the properties later on, so I've added the logic we have in other places as well and kept this value as a fallback in case any errors happen.